template/src/main/java/org/jboss/gwt/elemento/processor/HandlebarsParser.java

```diff
@@ -25,7 +25,7 @@
  */
 class HandlebarsParser {
```

Comment:
so this can no longer be called Handlebars parser... 😥

Reply:
i would like to say yes, you are right ..any suggestions for the name
Maybe PlaceHolderParser ?!

Reply:
hehe you already know, personally, I will maintain handlebar syntax... a 10years old project should have fixed this problem already.

Reply:
and this is pretty weird, I think you need to investigate further before start "randomly" changing the Elemento template syntax. Lucky for you I'm not the owner 😜. Se this discussion https://github.com/meteor/blaze/issues/71, actually web components already solves all the problems Elemento templating does, you should try some other alternatives like delegating the binding and templating to polymer instead of elemento. Or disabling elemento with a polymer template is used. Or something else, I have been playing with custom element for months before trying to make a pull request here, 😉 you can play a bit more with polymer integration. Anyways, the binding and templating conflict will produce more problems for sure, so at least discover it all. 👍

Reply:
i think you have a point here, but yet i think that i have tried to discuss the issue on gitter, IMO what i think should be in elemento is to keep it as simple as possible, for example this all expression evaluation may be left for other tools to handle, like polymer binding itself, or at least to leave it as optional, or to give the user of the api control over the template using hooks or something like that.

but with what we already have i cant continue using elemento for polymer templates, the handlebars are very common in polymer wc.

we still can keep this PR open until this issue get decided and good solution is provided, and i can build locally just to be able to continue testing with elemento.